pkg/operation/hybridbotanist/cloud_config.go

```diff
@@ -315,7 +315,7 @@ func (b *HybridBotanist) computeBootstrapToken() (secret *corev1.Secret, err err
 			bootstraptokenapi.BootstrapTokenDescriptionKey:      []byte("A bootstrap token generated by Gardener."),
 			bootstraptokenapi.BootstrapTokenIDKey:               []byte(tokenID),
 			bootstraptokenapi.BootstrapTokenSecretKey:           []byte(bootstrapTokenSecretKey),
-			bootstraptokenapi.BootstrapTokenExpirationKey:       []byte(metav1.Now().Add(90 * time.Minute).Format(time.RFC3339)),
+			bootstraptokenapi.BootstrapTokenExpirationKey:       []byte(metav1.Now().Add(24 * time.Hour).Format(time.RFC3339)),
```

Comment:
```suggestion
			bootstraptokenapi.BootstrapTokenExpirationKey:       []byte(metav1.Now().Add(48 * time.Hour).Format(time.RFC3339)),
```

Just to make sure that we don't run into any problem here if the reconciliation fails in the maintenance time window for some unrelated reason and doesn't reach this step.